### PR TITLE
Develop scenario unit tests

### PR DESCRIPTION
**Pull request by rossfauldsbjss**
scenario unit tests

### CODE REVIEW

File: packaged-sam.yaml [hunk 1]
```diff
@@ -1,120 +1,120 @@
-AWSTemplateFormatVersion: '2010-09-09'
```
Comment (danielchildsbjss) [inline packaged-sam.yaml]: I think we can delete this?

File: lib/utils.js [hunk 2]
```diff
@@ -256,6 +257,30 @@ const sendSessionDataToViews = function anonFunc(req, res) {
 		res.locals.scenario[key] = req.session.scenario[key];
 	});
 };
+exports.loadScenariosJsonFromFile = function loadFromFile(fileName, callback) {
```
Comment (danielchildsbjss) [inline lib/utils.js]: These can be more generic I think - don't need to mention Scenario

File: app/services/scenario.js [hunk 3]
```diff
@@ -1,23 +1,21 @@
...
-			if (response && response.statusCode === 200) {
-				this.scenarios = JSON.parse(body);
-			} else {
-				this.scenarios = [];
-			}
-		});
+	loadScenarios(scenarioJson) {
+		this.scenarios = [];
+		this.scenarios = JSON.parse(scenarioJson);
+		// request(scenarioUrl, (error, response, body) => {
```
Comment (danielchildsbjss) [inline app/services/scenario.js]: We can remove comments

**Comment by rossfauldsbjss** [comment]
All good points - will change now. Thanks.

**Pull request merged**